Source/utils/file_util.cpp

```diff
@@ -396,7 +400,11 @@ void RemoveFile(const char *path)
 	FILE *f = fopen(name.c_str(), "r+");
 	if (f != nullptr) {
 		fclose(f);
+#ifdef PSP
+		sceIoRemove(name.c_str());

```

Comment:
It'd be nice to add `remove` to the SDK instead. Filed https://github.com/pspdev/newlib/issues/7

Reply:
`remove` is now in the SDK! https://github.com/pspdev/pspsdk/pull/162